bootstrap.sh

```diff
@@ -7,6 +7,10 @@ bootstrap () {
      rm -rf autom4te.cache install-sh missing Makefile.in configure aclocal.m4 config.h.in
      autoreconf --force --install
     )
+
+    # uncomment PACKAGE_* variables to prevent redefine warnings
+    # because apache headers redefines these unfortunately to empty - but fortunately we do not use them
+    sed -i~ -e '/#undef PACKAGE_/ s|^/*|//|' -i config.h.in
```

Comment:
we define those, but apache headers (pulled later than our config.h) redefines them to empty. and i have seen nothing referencing them in our c source. we are using them other parts

Reply:
```
$ git grep PACKAGE_ | cat
Makefile.am:			   -DMODAUTHVAS_VERSION=\"$(PACKAGE_VERSION)\"
Makefile.am:	sed -e 's,[@]PACKAGE_VERSION[@],$(PACKAGE_VERSION),g' \
bootstrap.sh:    # uncomment PACKAGE_* variables to prevent redefine warnings
bootstrap.sh:    sed -i~ -e '/#undef PACKAGE_/ s|^/*|//|' -i config.h.in
configure.ac:	AC_MSG_ERROR([Your server does not support EAPI. $PACKAGE_NAME 3.6 and later require EAPI, please use $PACKAGE_NAME 3.5])
setup-mod_auth_vas4.in:PACKAGE_VERSION='@PACKAGE_VERSION@'
setup-mod_auth_vas4.in:        elif test x"$module_version" != x"$PACKAGE_VERSION"; then
setup-mod_auth_vas4.in:            expecting to see version '$PACKAGE_VERSION'.
```
No c source references the variables. I just removed them from config.h header, everywhere else they are still available. (Eg. Makefile.in) And in the c source they were redefined empty anyway. But it is just a warning fix so I can remove it if needed.